build.gradle

```diff
@@ -78,53 +78,55 @@ configurations {
 
 dependencies {
 
-    compile project(':email-client')
-    compile group: 'org.springframework.boot', name: 'spring-boot-starter-web', version: versions.springBoot
-    compile group: 'org.springframework.boot', name: 'spring-boot-starter-actuator', version: versions.springBoot
-    compile group: 'org.springframework.boot', name: 'spring-boot-starter-jdbc', version: versions.springBoot
-    compile group: 'org.springframework.boot', name: 'spring-boot-starter-aop', version: versions.springBoot
+  compile project(':email-client')
```

Comment:
better to do a separate PR for this sort of indenting, if this PR drags at all you have a high risk of conflicts if anyone touches this file at all